### ISSUE TITLE
Indent displays error in the wrong line

### ISSUE THREAD

**Issue by alexdmejias** [eslint/eslint#7880]
**Tell us about your environment**

* **ESLint Version:** 3.12.2
* **Node Version:** 4.7.0
* **npm Version:** 2.15.11

**What parser (default, Babel-ESLint, etc.) are you using?** default

**Please show your full configuration:**

```

{
    "rules": {
        "indent": ["error", 4]
    }
}


```

**What did you do? Please include the actual source code causing the issue.**

```js


define(function (require, exports, module) {
    var A = function () {
        return true;
    },
    B = function () {
        console.log('wasd')
    }
});


```

**What did you expect to happen?**
Nothing. According to my one rule, there shouldn't be any issues.

**What actually happened? Please include the actual, raw output from ESLint.**
eslint complains that there are indentation issues with line 3, even though it is indented 8 spaces, 4 spaces more that the line above it.

  3:9  error  Expected indentation of 12 spaces but found 8  indent
  6:5  error  Expected indentation of 8 spaces but found 4   indent
  7:5  error  Expected indentation of 8 spaces but found 4   indent

The issue is resolved if I only declare one variable per line.

```

define(function (require, exports, module) {
    var A = function () {
        return true;
    }
    var B = function () {
        console.log('wasd')
    }
});

```

**Comment by not-an-aardvark** [comment]
Thanks for the report, but this is working as intended -- the correct indentation is

```js
define(function (require, exports, module) {
    var A = function () {
            return true;
        },
        B = function () {
            console.log('wasd')
        }
});
```

When there is more than one variable declared in a `var` statement, the rule enforces that all the variables are offset by one indentation level, and all the assigned values are offset by one indentation level from the variables.

**Comment by platinumazure** [comment]
As I noted in chat, this can be fixed by using the "VariableDeclarator" option in configuration:

```json
{
    "rules": {
        "indent": ["error", 4, { "VariableDeclarator": 0 }]
    }
}
```

**Comment by kaicataldo** [comment]
Closing this issue as it looks like the question has been answered. Please feel free to visit us in the [ESLint Gitter](https://gitter.im/eslint/eslint) if you have any other issues!
